frontend/app/components/Thermometer.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import styles from "./styles/Thermometer.module.scss";
+
+type Props = {
+  tasks: string;
```

Comment:
ich denke mal das ist noch nicht fertig, ich würde aber die tasks in total numbers rein geben und prozentual in der Komponente ausrechnen